packages/graphql/src/scripts/2025-01-15_cleanup_solutions_feedbacks_elements.ts

```
@@ -0,0 +1,348 @@
+            // await prisma.element.update({
+            //   where: { id: element.id },
+            //   data: {
+            //     options: {
+            //       ...element.options,
+            //       hasSampleSolution: false,
+            //       solutionRanges: undefined,
+            //     },
+            //   },
+            // })
+
+            missingNumericalSampleSolution++
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)